guardian/recipes
# Enable merging steps

## What does this change?
Allow merging multiple steps to help with the recipe editing process.

https://github.com/guardian/recipes/assets/47318984/cde36ae4-a050-455c-b578-c1de66184e45


## How to test
- Try merging steps by selecting multiple and clicking the 'merge steps' button. 
- Steps in any combination (sequential or not) should work and the step numbers should be updated accordingly. 
- Recipe should save without error after the change and reloading the page should show the updated status.

## How can we measure success?
